clusterloader2/pkg/measurement/util/runtimeobjects/replicaswatcher.go

```diff
@@ -40,9 +40,11 @@ const (
 // ReplicasWatcher is a struct that allows to check a number of replicas at a given time.
 // Usage:
 // var rw ReplicasWatcher = (...)
-// if err := rw.Start(stopCh); err != nil {
-//   panic(err);
-// }
+//
+//	if err := rw.Start(stopCh); err != nil {
```

Comment:
I'm not sure if I understand this change :)

this will result in code snipped with weird formatting (some part indentend, some not).

Reply:
Probably some automatic formatting in vs code or sth similar :)

Reply:
I was thinking about that too, but in theory var can be defined somewhere else (e.g. this snippet can be inside some nested statement)
So I think it's fine :)

[Although I don't have strong opinion here :) ]

Reply:
vscode magic, somehow some code has tabulations